lib/diagnostic.md

```diff
@@ -3,7 +3,12 @@
 Using `inspect element`, write four selectors found on the main page for this GitHub repo.
 
 <!-- your answer starts here -->
-
+```
+<div class="facebox" id="facebox">
+<div class='header header logged in true>
+<div id='start-of-content' class='accessability-aid'>
+<div id='js-flash-container'>
+```
```

Comment:
this is an opening div tag: `<div class="facebox" id="facebox">`
this is a selector: `facebox`